docs/coding-projects/hello-world.md

```diff
@@ -0,0 +1,384 @@
+---
+title: Hello world, revisited
+description: "Start coding with the Pact smart contract programming language by creating a 'Hello, World!' project."
+id: hello-world
+sidebar_position: 2
+layout: full
+tags:
+  [
+    'pact',
+    'hello world',
+    'tutorials',
+    'hello world tutorial',
+    'pact hello world',
+    'smart contract deployment',
+  ]
+---
+
+# Hello, World! revisited
+
+In the [Quick start](../quickstart), you were introduced to a simple `greeting` module. 
+Modules provide the basic foundation for all Pact smart contracts. 
+In fact, the term `module` is a reserved keyword that you use to define and install a module with a specified _name_, 
+
+The `greeting` module defined a single function—one very much like the traditional Hello, World! program you see when learning any new programming language.
+
+In the `00-hello-world` project folder, you'll take a closer look at the `hello-world` module, extend its functionality, and deploy it on the local development network.
+
+## Before you begin
+
+Before starting this project, verify your environment meets the following basic requirements:
+
+- You have a GitHub account and can run `git` commands.
+- You have installed the Pact programming language and command-line interpreter.
+- You have installed the `kadena-cli` package and have a working directory with initial configuration settings.
+- You have a local development node that you can connect to that runs the `chainweb-node` program, either in a Docker container or on a physical or virtual computer.
+
+If you have these basics covered, you're ready to go.
+
+## Get the starter code
+
+To get started:
+
+1. Open a terminal shell on your computer.
+
+2. Clone the `pact-coding-projects` repository by running the following command:
+   
+   ```bash
+   git@github.com:kadena-docs/pact-coding-projects.git
+   ```
+
```

Comment:
The base directory also contains a `helloWorld.pact` file, maybe it should be removed or renamed (https://github.com/kadena-docs/pact-coding-projects/blob/main/hello-world.pact). I think it will cause confusion.